docs/README.md

```diff
@@ -67,14 +67,14 @@ following the instructions.
         $ cd /path/to/marathon-gh-pages
         $ git checkout gh-pages
 
-3. Copy the contents of the "docs" directory in master to the root of your
+3. Check out the appropriate release branch, then copy the contents of the "docs" directory in master to the root of your
```

Comment:
@janisz why is step 2 redundant? gh-pages still lives on a separate branch, but we sync from the release branch, instead of master.